src/middleware/lib/perform-and-respond.js

```diff
@@ -0,0 +1,16 @@
+import formatMongooseError from './format-mongoose-error';
+
+export default async function(promise, res) {
+    try {
+        let data = await promise;
+        if (_.isUndefined(data)) {
+            res.sendStatus(204);
```

Comment:
https://benramsey.com/blog/2008/05/http-status-204-no-content-and-205-reset-content/